### PR TITLE
Edited Modal resposivness, added Web Ilustration

### PR DESCRIPTION
**Pull request by guyesh**
Fixed a bug in the modal responsiveness.
Added web illustration to modal.
**Solution to the android problem, keyboard hides the input field:**
Changed android modal css position to sticky - makes the modal jump up when trying to write to the input field.
made the image disappear when clicking the 'enter location manually' button( on android)

### CODE REVIEW

**Comment by guytepper** [comment]
Thanks @guyesh! I'll review it soon. 
Meanwhile, some notes on the illustration: 
- The filesize is 44kb, which is a lot. Try to use [SVGOMG](https://jakearchibald.github.io/svgomg/) to reduce the file size, but if it doesn't make it go under <10kb~ - use jpg & webp to display it.
- Change the file name to something descriptive, such as `welcome-illustration.svg`
- If Lena haven't talked to you yet, please send her a message on discord (`@Lena`)

**Comment by guytepper** [comment]
The manual address button is hidden initially (page load) on iPhone 7 Plus:


Maybe placing the modal slightly above the header can help?

**Comment by guyesh** [comment]
sure! will be done.

**Comment by guyesh** [comment]
**made the modal scrollable when the screen is shorter than the modal**
this solved the android problem( that the keyboard hides the input field).
also on some short landscape screens, the modal was cut off by the bottom of the screen. the scrollable modal solved the problem.
**Added a low-resolution image (6KB) to appear when the high-resolution one(28KB) is being loaded**, thus the user doesn't need to wait for the image but also doesn't lose quality.

**Comment by guyesh** [comment]
**IPHONE 11 MAX Image:** 
IPHONE 11 MAX Image
**samsung note 9 Images:** 
samsung note9
samsung note9

File: src/components/Modal/Modal.js [hunk 1]
```diff
@@ -43,12 +43,28 @@ function Modal({ isOpen, setIsOpen, coordinates, setCoordinates }) {
     return () => clearTimeout(timeout);
   }, [addressInputDisplay]);
 
+  const [ImageLoaded, setImageLoaded] = useState(false);
```
Comment (guytepper) [inline src/components/Modal/Modal.js]: I think this is an overkill. We can use responsive images (`<picture>` tag) to solve most performance issues :)

File: src/components/Modal/Modal.js [hunk 2]
```diff
@@ -43,12 +43,28 @@ function Modal({ isOpen, setIsOpen, coordinates, setCoordinates }) {
...
   return (
     <ModalWrapper isOpen={isOpen}>
       <ModalContentWrapper>
+        {/* image "thumbnail" lower resolution image until the high resolution one loaded"*/}
+        <img
+          src="welcome-illustration-lowres.webp"
+          style={{ width: 250, height: 176.7, display: ImageLoaded ? 'none' : '' }}
+          alt="אילוסטרציה של הפגנה"
```
Comment (guytepper) [inline src/components/Modal/Modal.js]: We don't really need a screen reader to explain what this image is, we can just put empty alt tags.

File: src/components/Modal/Modal.js [hunk 3]
```diff
@@ -43,12 +43,28 @@ function Modal({ isOpen, setIsOpen, coordinates, setCoordinates }) {
     return () => clearTimeout(timeout);
   }, [addressInputDisplay]);
 
+  const [ImageLoaded, setImageLoaded] = useState(false);
   return (
     <ModalWrapper isOpen={isOpen}>
       <ModalContentWrapper>
+        {/* image "thumbnail" lower resolution image until the high resolution one loaded"*/}
+        <img
```
Comment (guytepper) [inline src/components/Modal/Modal.js]: I א

**Review by guytepper — COMMENTED** [review]
Thanks! Great work, and bravo for finding the keyboard overflow issue. 
I'm leaving a few notes that I've already fixed :-)

[Ended up going with `<picture>` to improve loading time instead of async image load.]

**Pull request merged**